setup_cfg_fmt.py

```diff
@@ -175,6 +175,9 @@ def _tox_envlist(setup_cfg: str) -> Generator[str, None, None]:
                 yield env
 
 
+TOX_ENV = re.compile('py(\\d)(\\d+)')
```

Comment:
I think this would be more clear as a raw string `r'py(\d)(\d+)'`

Reply:
Sure